package.json

```diff
@@ -354,6 +354,236 @@
+						"description": "%reactNative.snippets.debugAndroid.description%",
+						"body": {
+							"name": "Debug Android (Hermes)",
+							"cwd": "^\"\\${workspaceFolder}\"",
+							"type": "reactnativedirect",
+							"request": "launch",
+							"platform": "android"
+						}
+					},
+					{
+						"label": "React Native (Hermes): Attach to packager",
+						"description": "%reactNative.snippets.attachPackager.description%",
+						"body": {
+							"name": "Attach to packager (Hermes)",
+							"cwd": "^\"\\${workspaceFolder}\"",
+							"type": "reactnativedirect",
+							"request": "attach"
+						}
+					}
+				],
+				"configurationAttributes": {
+					"attach": {
+						"required": [
+							"cwd"
+						],
+						"properties": {
+							"program": {
+								"type": "string",
+								"description": "%reactNative.attach.program.description%"
+							},
+							"cwd": {
+								"type": "string",
+								"description": "%reactNative.attach.cwd.description%",
+								"default": "${workspaceFolder}"
+							},
+							"sourceMaps": {
+								"type": "boolean",
+								"description": "%reactNative.attach.sourceMaps.description%",
+								"default": false
+							},
+							"sourceMapPathOverrides": {
+								"type": "object",
+								"description": "%reactNative.attach.sourceMapsPathOverrides.description%"
+							},
+							"trace": {
+								"type": "string",
+								"enum": [
+									"verbose",
+									"log",
+									"warn",
+									"error",
+									"stop"
+								],
+								"default": "log",
+								"description": "%reactNative.attach.trace.description%"
+							},
+							"address": {
+								"type": "string",
+								"description": "%reactNative.attach.address.description%",
+								"default": "localhost"
+							},
+							"port": {
+								"type": "string",
+								"description": "%reactNative.attach.port.description%",
+								"default": "8081"
+							},
+							"remoteRoot": {
+								"type": "string",
+								"description": "%reactNative.attach.remoteRoot.description%",
+								"default": null
+							},
+							"localRoot": {
+								"type": "string",
+								"description": "%reactNative.attach.localRoot.description%",
+								"default": "${workspaceFolder}"
+							},
+							"skipFiles": {
+								"type": "array",
+								"description": "%reactNative.attach.skipFiles.description%",
+								"items": {
+									"type": "string"
+								},
+								"default": []
+							},
+							"debuggerWorkerUrlPath": {
+								"type": "string",
+								"description": "%reactNative.attach.debuggerWorkerUrlPath.description%",
+								"default": "debugger-ui/"
+							},
+							"launchActivity": {
+								"type": "string",
+								"description": "%reactNative.attach.launchActivity.description%",
+								"default": "MainActivity"
+							}
+						}
+					},
+					"launch": {
+						"required": [
+							"cwd",
+							"platform"
+						],
+						"properties": {
```

Comment:
Please, remove all unverified properties. We don't have yet information about how we support them for teh new debugger, so let's add them later as more information will be available.